webpack.config.shared.js

```diff
@@ -57,7 +57,11 @@ module.exports = {
     exclude: /node_modules/,
     loader: 'babel-loader',
     query: {
-      "presets": ["latest", "react"],
+      "presets": [
+        ["env", { "modules": false }],
```

Comment:
take a look [here](https://medium.com/webpack/webpack-2-and-beyond-40520af9067f#.8relf4lln) and [here](https://babeljs.io/docs/plugins/preset-env/#options) for info - basically what it does is cause Babel to not transform any `import` or `export` statements to `require` statements. This should allow for tree shaking to work, although (as [this issue](https://github.com/webpack/webpack/issues/2867) points out) tree shaking is pretty broken right now.